akka-remote/src/test/scala/akka/remote/artery/RemoteDeploymentSpec.scala

```diff
@@ -47,10 +47,11 @@ object RemoteDeploymentSpec {
       case p: Props ⇒
         sender() ! context.actorOf(p)
 
-      case (p: Props, firstMsg: Int) ⇒
+      case (p: Props, numMessages: Int) ⇒
         val child = context.actorOf(p)
         sender() ! child
-        child.tell(firstMsg, sender())
```

Comment:
Reproduced by adding sleep(200) here, which would mean that the test sent it messages before the `firstMsg`.

Instead, send all messages here.